roles/ceilometer-data/tasks/main.yml

```diff
@@ -0,0 +1,19 @@
+---
+- name: install ceilometer compute service
+  upstart_service: name=ceilometer-agent-compute
+                   user=ceilometer
+                   cmd=/usr/local/bin/ceilometer-agent-compute
+  notify:
+    - restart ceilometer services
+
+- name: pip install libvirt-python in venv on compute
+  pip: name=libvirt-python virtualenv="{{ ceilometer.virtualenv }}"
+  notify:
+    - restart ceilometer services
```

Comment:
Why not do this bit as part of the source install? Does it harm anything to have libvirt-python in the venv on all the hosts?

Maybe this bit was copied from another role, where we have to install this big into the /package/ virtualenv because some install time compilation is required. In those cases, they are conditioned with a `when` statement.